src/maintagent/agent/maintagent.h

```diff
@@ -33,6 +33,9 @@
 #include <libfossology.h>
 #define FUNCTION
 
+/**
+ * Maximum buffer to use
```

Comment:
actually, ok, but not so much informative. I understand that there is not more information currently, but ...